### ISSUE TITLE
i can't open

### ISSUE THREAD

**Issue by mikolps** [cheat-engine/cheat-engine#1310]
i cannot open cheat engine 7.1. when i start it, nothing happens. i have tried everything the cheat engine forums told me to do. nothing helped. please help me.

**Comment by cheat-engine** [comment]
have you tried uninstalling badlion using their uninstaller (so NOT just delete files you think are related) and then restart windows (Not shut down and back on, actual restart)

**Comment by mikolps** [comment]
i tried all that and nothing happened. i have tried the windowsrepair.exe and nothing heppened.

**Comment by cheat-engine** [comment]
in a command prompt do a `driverquery /v > bla.txt` . It will create a bla.txt file in the directory you execute that command.  Post it here see we can see what may be blocking you

**Comment by mikolps** [comment]
i did that and the result is on the pastebin i linked. https://pastebin.com/5sLbwjNu
hope this helped!

**Comment by cheat-engine** [comment]
```BadlionAntic BadlionAnticheat       BadlionAnticheat       Kernel        System     Running    OK         TRUE        FALSE        4,096             3,813,376   0          5/26/2020 2:58:48 PM   \??\C:\WINDOWS\system32\drivers\BadlionAnticheat 4,096      ```

You still have the badlion anti cheat running on your system

Uninstall it, and restart windows

If uninstalling is an issue then execute this in a command prompt that is running with Administrator access
```
net stop BadlionAntic
net stop BadlionAnticheat
sc delete BadlionAntic
sc delete BadlionAnticheat
```

and do not forget to restart windows.  (Again, do NOT shut down and back on, do an actual restart)

**Comment by mikolps** [comment]
thank you so much! it now works!

**Comment by Janmm14** [comment]
I personally would not classify BadlionAnticheat as malware.
Sure, the blockage of cheat engine starting is kind of invasive, but still.

**Comment by cheat-engine** [comment]
It doesn't notify the user it's blocking CE
It runs when the user doesn't want it to run
It runs even when booted in safe mode
You can not easily delete it

Those are malware flags if you ask me

**Comment by animeeaex** [comment]
none of that worked for me, i want to start it and just nothing happens, i never had badlion installed btw but still tried that

**Comment by cheat-engine** [comment]
check malwarebytes or another badly designed anti virus that doesn't look at contents
Does the installer run, or also nothing(if nothing for the installer it's windows defender), and have you done the `driverquery /v > bla.txt` thing ?  Upload the bla.txt here it may give a clue what else it can be

**Comment by git42leech** [comment]
https://pastebin.com/jdfjhtTU

CheatEngine71.exe will not run, Windows Defender is completely disabled.

**Comment by cheat-engine** [comment]
it looks like windows defender is still on.
Try disabling 'reputation based protection '

**Comment by CyberHUN0** [comment]
I cannot open cheat engine 7.1. when i start it, nothing happens. My bla.txt : https://pastebin.com/M1pjdLd1

**Comment by cheat-engine** [comment]
`BadlionAntic BadlionAnticheat       BadlionAnticheat       Kernel        System     Running    OK         TRUE        FALSE        4˙096             3˙813˙376   0          2020. 05. 26. 14:58:48 \??\C:\WINDOWS\system32\drivers\BadlionAnticheat 4˙096      `

you have been infected by the badlion malware 
check the post above on how to clean your system

**Comment by CyberHUN0** [comment]
Thank you so much for tips! But I used this command :  sc config BadlionAnticheat start= disabled

**Comment by CyberHUN0** [comment]
The point is that it worked so thank you!

**Comment by divinity76** [comment]
for the record, use `shutdown -r -f -t 1` , not just `shutdown -r` , not sure of the specifics, but lots of programs can block windows from restarting if you issue `shutdown -r` , but few programs block windows from restarting if you use `shutdown -r -f -t 1`

**Comment by fade2gray** [comment]
@divinity76 
Users are asked to use **_Restart_**, and not **_Shutdown_**, after uninstalling badlion for a very specific reason.

**Comment by divinity76** [comment]
@fade2gray yeah.. the `-s` flag means `shutdown` , the `-r` flag means `restart` , the `-f` flag means `force`, and the `-t 1` flag means `in 1 second`

lots of programs will prevent the restart if the `-f` flag is omitted

**Comment by Janmm14** [comment]
Badlion 3.0 allows users to install and uninstall the BAC driver in its GUI.

**Comment by greatniller** [comment]
cheat engine will not download ihave the launcher but nothing happens after that

**Comment by PeachMangoes** [comment]
After disabling (Off by default) the DEP(data execution prevention) on exploit protection windows, the CE7.1 worked for me.

**Comment by jacklolidk** [comment]
i uninstalled the badlion "malware" and still, it doesnt work, 7.2
bla file here: 
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/6662058/bla.txt)

**Comment by cheat-engine** [comment]
No error at all, or an access denied error? Ifbaccess denied you'll have to disable reputation based protection

**Comment by jacklolidk** [comment]
nope

**Comment by jacklolidk** [comment]
but it did try to make me restart while installing, do i have to restart?

**Comment by cheat-engine** [comment]
Yes.  If CE's installer tells you you need to restart, do so. (Not shut down and back on, actual restart)

**Comment by jacklolidk** [comment]
ok

**Comment by jacklolidk** [comment]
yeeep

**Comment by gklogamer26** [comment]
I am having an error, it is showing
Windows cannot access the specified device, path, or file. You may not have the appropriate permissions to access them.

**Comment by gklogamer26** [comment]
Well, nvm it was windows defender just messing around :/

**Comment by Bigpapa114** [comment]
I don't have badlion and when i run the 7.1 setup  it disappears all my defender option are off also rbp is off. I am able to install 7.2 but not 7.1 and it just happened after I installed and uninstalled 7.2. earlier it was working fine
my bla file.
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/6681748/bla.txt)



DO HELP!!

**Comment by cheat-engine** [comment]
disabke reputation based protection

**Comment by WRCUno** [comment]
i don't have badlion installed and i dont have rdp enabled... having the same scenario as bigpapa114
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/6729740/bla.txt)
![Capture](https://user-images.githubusercontent.com/86634664/123717635-19569080-d832-11eb-9b4b-99c10b7bfb5f.PNG)

**Comment by cheat-engine** [comment]
Have you already restarted windows? (not shut down amd back on, actual restart)

**Comment by WRCUno** [comment]
I figured it out, I had to download the chest engine setup

**Comment by TheCamperGamer** [comment]
i cant open it, nor can i disable windows defender. i dont have the "virus and threat protection" page so i cant disable it. ive tried downloading registries to enable "virus and threat protection" page but nothing works.

**Comment by VeritasDL** [comment]
> i cant open it, nor can i disable windows defender. i dont have the "virus and threat protection" page so i cant disable it. ive tried downloading registries to enable "virus and threat protection" page but nothing works. 

Sounds like a virus messed with your security settings, my old install of windows 10 did the same when a sibling installed some BS Roblox "hack"

**Comment by Diamondhawk8** [comment]
Hey there, having an issue where I run the installer, it says it installed, but Cheat Engine does not open and is nowhere to be found, ive disabled my antivirus and done the  reputaion based protection thing but it still wont work. heres the driverquery https://pastebin.com/LUH1aFN3

**Comment by 5tyrtey** [comment]
i cant get cheat engine to open

**Comment by minkycarcass** [comment]
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/6926672/bla.txt)
![Screenshot 2021-08-03 144521](https://user-images.githubusercontent.com/88403186/128076624-c0445ee0-1ed5-4405-b77d-6333ca7dcaf0.png)


Im unable to get CE running. The setup opens and asks for language, then stalls and shows a constant loading bar. I've already disable reputation based protection and dont think i have Badlion Malware. Any Advice?

**Comment by cheat-engine** [comment]
Check for adblocking software/firewalls that block downloads from known ad-serving domains.  The CE Downloadersetup will download the latest CE install from one of those domains

**Comment by minkycarcass** [comment]
I've completely disabled windows defender and its firewall and for some reason it keeps not responding on me.

**Comment by samukaka** [comment]
não tenho badlaion e mesmo assim não abre podem me ajudar por favor ?

**Comment by samukaka** [comment]
meu bla.txt 
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/6927466/bla.txt)

**Comment by sanikdah** [comment]
I have the CE 7.2 Installer, runs, takes like, .5 seconds on the "Downloading additional files" section, then says that it finishes, yet no trace of CE aside from the installer is on my system, HELP!

**Comment by gjud1234** [comment]
hello can not download it I need version on my computer will help me please

**Comment by oofasaur** [comment]
Help, Cheat Engine won't open. Windows Defender is "protecting" my system and deleting the file. 
Here's the bla.txt: https://pastebin.pl/view/69cbb0fd
(it's in portuguese)

**Comment by Halim-11** [comment]
hello guys i want to communication with cheat engian in discored

**Comment by DRODRAGROA** [comment]
> @fade2gray yeah.. the `-s` flag means `shutdown` , the `-r` flag means `restart` , the `-f` flag means `force`, and the `-t 1` flag means `in 1 second`
> 
> lots of programs will prevent the restart if the `-f` flag is omitted

I have the PUADIManager.Win32 / OfferCore on me when I install the cheat engine it asks me if I want to install it so that I can open the cheat engine, does this PUADI manager belong to it?

**Comment by DRODRAGROA** [comment]
> disable reputation based protection

I have the PUADIManager.Win32 / OfferCore on me when I install the cheat engine it asks me if I want to install it so that I can open the cheat engine, does this PUADI manager belong to it??????

**Comment by DRODRAGROA** [comment]
I'm desperate to install the application here.

**Comment by Qboi123** [comment]
> I'm desperate to install the application here.

Maybe try installing it in a Virtual Machine first.

**Comment by DRODRAGROA** [comment]
i have done with ignito mode have delete it and allow ower the windows security aktion.

**Comment by cheat-engine** [comment]
yes, PUA is safe.  You just have to read and decide if you want the advertised offer or not.

**Comment by Halim-11** [comment]
so can i use it now for ROK ?

**Comment by arsninja** [comment]
Whenever I download and try to launch Cheat Engie I get the error "windows cannot access the specified device, path or file. You may not have the appropriate permissions to access the item." How can I fix this?

**Comment by cheat-engine** [comment]
@arsninja in the notification center find the block notification and allow it, or disable reputation based protection in windows.  And if you have AVG turn of some obscure setting somewhere

**Comment by arsninja** [comment]
@cheat-engine what is AVG

**Comment by lucaizz** [comment]
im having the same issue, i dont know what can be... I can download it, install it but when i click on the program it dont execute, also i never had the badlion, my windows defender is not on and i dont have any other anti virus, and yes, i disable the  reputation based protection...

**Comment by lucaizz** [comment]
anyone can help me?

**Comment by deadriver35** [comment]
can any one help me? i my cheat engine 7.3 will shut down right after it open. and any website i open that have the word "cheat engine" will automatic shutdown the whole browser. i have to use my phone to ask this.

**Comment by Eluision** [comment]
hi cheat engine i got a similar problem and do not understand a bit of this bla txt
https://pastebin.com/7yNx2Qqz

**Comment by Eluision** [comment]
ok ive managed it. i added to  the ignored list of windowsdefender, sry if its not called like this in english os

**Comment by fade2gray** [comment]
@Eluision Well done, it's a pity others can't figure that out for themselves.

**Comment by Lima-DF** [comment]
It does not open.
Here is the bla.txt:
https://pastebin.com/UNTdmFQB

**Comment by metalJeebus** [comment]
> It does not open. Here is the bla.txt: https://pastebin.com/UNTdmFQB

Same issue here, followed all the advised steps and rest everything a dumb amount of times and still no closer to getting it running

**Comment by fade2gray** [comment]
Simple error free method to download and install Cheat Engine.

#1954

**Comment by metalJeebus** [comment]
> Simple error free method to download and install Cheat Engine.
> 
> #1954

I've attempted the exclusion method a handful of times since since my initial post and that doesnt work, the launcher just will not open

**Comment by fade2gray** [comment]
@metalJeebus

Sorry, I can't reproduce the same issue. The video was made on a clean Win 10 virtual machine with no security other than that supplied by Windows. Do you use any other antivirus software, if so and it has a free or trial version, perhaps I could try it? Otherwise, the only thing I can suggest is trying to remove any evidence of your previous CE installation by clearing your temp files and installation directory, or even reverting to a restore point made prior to the installation.

**Comment by hansbrill** [comment]
Is it possible to see what happens next in online casino games?

**Comment by ThomasTheTankOG** [comment]
Hey so I'm trying to install from the website and it will show that it's installed before google will say download failed virus detected. I don't have bad lion and I've tried turning off google protection just for this but still no luck. My only antivirus on my computer is malwarebytes but I can still allow the download even with this antivirus. Any help is very appreciated and thank you in advance

**Comment by StupidRepo** [comment]
> It doesn't notify the user it's blocking CE
>It runs when the user doesn't want it to run
>It runs even when booted in safe mode 
> You can not easily delete it
>
> Those are malware flags if you ask me

Those might be malware flags to you but for me, I see it as a good feature because it's just Badlion's way to stop people from hacking their client. It's like Fortnite's Anti-Cheat.

**Comment by THEBOZ7** [comment]
I need help with Cheat engine, basically I can open the application but I can't interact with it. Any ways to fix this?

**Comment by fade2gray** [comment]
@THEBOZ7  I'd create a separate issue for your problem, where it will more likely be noticed by someone who knows how to address it.

**Comment by jasonmp85** [comment]
@cheat-engine I logged in just to say I wouldn't run around calling anti-cheat software "malware" while you're sitting here (a) writing software that patches binaries in-memory, (b) bundling said risky software with a ton of ad shit during install, and (c) directing people to disable pieces of Windows security.

Your site doesn't even say what your software DOES but instead just has a huge download button atop it. What did you call it? "malware flags"? Yeah.

**Comment by Hellrais3r** [comment]
**CE 7.4 doesn't work for M1 Macs**, like the new MacBook Pro M1. No scans at all. I still using CE 7.3, that is partially working (no Pointer scans etc.). CE 7.3 and 7.4 only work with old Intel CPU. Thank you all.

**Comment by Painisthegame** [comment]
can you tell me what my problem is?
[bla.txt](https://github.com/cheat-engine/cheat-engine/files/8028511/bla.txt)

**Comment by fade2gray** [comment]
@Painisthegame Need more detail - what are you doing, when are you doing it and what's happening as a result, preferably in a new thread. Include images/screenshot of any errors, or a precise description.

**Comment by Painisthegame** [comment]
> @Painisthegame Need more detail - what are you doing, when are you doing it and what's happening as a result, preferably in a new thread. Include images/screenshot of any errors, or a precise description.

@fade2gray
I have followed all the instructions and it comes up with an error saying I didn't have the permissions and when I run it as administrator it says it's a virus

Could you please read the bla.txt file and find what the problem is?
I would really appreciate it

**Comment by fade2gray** [comment]
@Painisthegame If you can't read and follow what is asked of you, how will you be able to follow instructions on how to fix the problem?

**Comment by cualquiercosa327** [comment]
Hello.I am trying to install Cheat Engine on my windows 10 but i am unable.I have a folder where windows doesnt look the files,also i have disable fully Windows defender but i can not install.
When i tried this message appears:
![Captura](https://user-images.githubusercontent.com/16308276/156003284-d0ccc10a-010c-46ce-9972-6a27b5ca6e9c.JPG)
 
If i tried older versions it ask me permission to change things on the device,i press Yes,but it open on the task bar (as it happens when i run any program) but it is transparent and it dissapears after 2 seconds.

This is my bla file.
Thanks and excuse me

[https://pastebin.com/5sLbwjNu](url)
![Captura](https://user-images.githubusercontent.com/16308276/156003816-afa963bf-bbf1-4a88-b38f-9f8e559b3baf.JPG)

**Comment by fade2gray** [comment]
@cualquiercosa327 Do you use a VPN? See [this](https://github.com/cheat-engine/cheat-engine/issues/2020#issuecomment-1032614034).

**Comment by cualquiercosa327** [comment]
> @cualquiercosa327 Do you use a VPN? See [this](https://github.com/cheat-engine/cheat-engine/issues/2020#issuecomment-1032614034).

Hi.No i dont use. I download on one pc ,put it on a flash memory an install it on a computer which has no internet conexion

**Comment by fade2gray** [comment]
You can't install without a live Internet connection - it is an online installer.

**Comment by pop-pro** [comment]
When CE is turned on, it displays
Cheat Engine won't launch (0x000007b) problem.

**Comment by fade2gray** [comment]
@pop-pro Try running `windowsrepair.exe` from your CE installation directory. Failing that, do a Google search for `Windows 0x000007b`.

**Comment by SHORTUDE** [comment]
![image](https://user-images.githubusercontent.com/104080994/164245129-43ee3197-cfe9-4c3a-b31b-bee66d94ca18.png)
Every time i try to download the cheat engine this just pops up I've tried multiple browsers but it still doesn't work

**Comment by fade2gray** [comment]
Are you using a VPN or proxy? See #2099

**Comment by SHORTUDE** [comment]
I am not but I have eset but I turned it off when I downloaded the cheat engine but the same error popped up

**Comment by Kamran3456** [comment]
when I download Cheat Engine it says that Failed- Insufficient permissons

**Comment by Herold11** [comment]
cheat engine isn't working because of network

**Comment by DavedZrod** [comment]
![image](https://user-images.githubusercontent.com/90824958/177244530-108a7842-4c5e-4e31-8d13-5a81ccfa9464.png)
i use Kaspersky Antivirus Free (even turned it off) so this isnt an option https://github.com/cheat-engine/cheat-engine/issues/1310#issuecomment-671385081 i also have Comodo firewall, ofc it blocked it (kaspersky didn't) so i closed it. 
i have never installed badlion, i did previusly have vanguard but uninstalled using the installer + the file directory.
The installer only runs for a few secs seeing on task manager and it closes then shows Error
https://github.com/cheat-engine/cheat-engine/issues/2010 didnt help either

**Comment by fade2gray** [comment]
@DavedZrod 

> i use Kaspersky Antivirus Free (even turned it off)

Kaspersky will continue to run in kernel mode.

https://github.com/cheat-engine/cheat-engine/issues/941#issuecomment-542851056
https://github.com/cheat-engine/cheat-engine/issues/938#issuecomment-590568380

**Comment by DavedZrod** [comment]
Uninstalled kaspersky and restarted, followed along #1954 but only managed to get to the point where it asks to run (3:18 on the video) "Open file - Security Warning" when i press run it didn't run at all no loading bar

**Comment by fade2gray** [comment]
@DavedZrod 

Open a command prompt and enter `Driverquery /v | findstr Kaspersky`.
If you see anything listed, it means Kaspersky is still running in kernel mode - starts each time you restart Windows.

See https://support.kaspersky.com/KESWin/11.3.0/en-US/128004.htm

**Comment by DavedZrod** [comment]
no results

**Comment by fade2gray** [comment]
@DavedZrod 
If you've created an exception for your temp directory in Defender with Real-time protection disabled etc, and you're not using a VPN, proxy or a PiHole, it seems it might be your ISP blocking the final connection and subsequent file download. If your PC has wifi, you could try connecting to a hotspot on a mobile device and use the mobile-data for the final download.

The only other things I can suggest are, try installing to another PC and copying the installation folder over, or compiling your own.

**Comment by DavedZrod** [comment]
I followed the build instructions and it runs, Thanks. although i have some questions, do i have to always run the cheatengine-x86_64.exe at .../Cheat engine/bin or can i move it elsewhere? Do i only need the exe for it to run?

**Comment by DarkSoulsEnjoyer** [comment]
Hello, I downloaded Cheat Engine from the Dark Souls discord server, but when I try opening Cheat Engine it doesn't open. It doesn't say anything about any error or anything either. I've completely uninstalled Mcafee from my pc, as well as making cheat engine and the .ct file and exception on the windows av, so im just really confused on what could possibly be the problem, please help, many thanks.

**Comment by cheat-engine** [comment]
`I downloaded Cheat Engine from the Dark Souls discord server`
go to file properties and verify that the signature is by enginegame or cheatengine

if there is no signature or it"s something else, you likely have a fake file

**Comment by WillXProject** [comment]
@cheat-engine, Badlion isn't malware, although it certainly seems like it, especially since it injects a driver/service into the system, although this is the same as Valorant's anticheat, it blocks Cheat Engine among other programs and is very invasive, it works in safe mode (due to the driver) and does not notify the user of what it does in the background, luckily, the way to remove it is as simple as the following in an Elevated/Administrator command prompt:
```
sc delete BadlionAnticheat
shutdown /r /f /t 0
```
Badlion is actually a legitimate Minecraft Client (Java Edition), although not supported by Mojang and Microsoft, BAC (BadlionAntiCheat) is the service doing all the shady stuff, I don't know why they won't bother to tell their users and don't give an option to turn it off, other than uninstalling the client, shady as f*ck.

**Comment by akustom** [comment]
hey, i downloaded cheat engine but it does nothing, the same as everybody else can you check the problem, here is my bla.txt https://pastebin.com/TDbgsYeE

**Comment by fleak1** [comment]
> i cannot open cheat engine 7.1. when i start it, nothing happens. i have tried everything the cheat engine forums told me to do. nothing helped. please help me.

cheat wont open i dont have batlion all antivirus or of its making mad asf

**Comment by xuloxe** [comment]
please help me when i install CE that tells me "a device attached to the system is not functioning" I don't understand anything I've tried lots of things and nothing works please help me

**Comment by cheat-engine** [comment]
disable "reputation based protection"

**Comment by xuloxe** [comment]
it does not work that tells me again "a device attached to the system is not functioning"

**Comment by cheat-engine** [comment]
then also disable bitdefender, windows defender, webroot  (note that for the last one you do need to restart windows)

**Comment by xuloxe** [comment]
THANK YOU it was webroot I had deactivated it but had to delete it thank you and have a good day

**Comment by ctiKyo** [comment]
> `BadlionAntic BadlionAnticheat BadlionAnticheat Kernel System Running OK TRUE FALSE 4,096 3,813,376 0 5/26/2020 2:58:48 PM \??\C:\WINDOWS\system32\drivers\BadlionAnticheat 4,096 `
> 
> You still have the badlion malware running on your system
> 
> Uninstall it, and restart windows
> 
> If uninstalling is an issue then execute this in a command prompt that is running with Administrator access
> 
> ```
> net stop BadlionAntic
> net stop BadlionAnticheat
> sc delete BadlionAntic
> sc delete BadlionAnticheat
> ```
> 
> and do not forget to restart windows. (Again, do NOT shut down and back on, do an actual restart)
> 
> command prompt : `shutdown /r` might work

My problem is similar to this. Please help me. Link : https://pastebin.com/uwP2DFTM

**Comment by C4ft4w** [comment]
Bantu download chat-angine buat system saya

**Comment by C4ft4w** [comment]
Halo, saya ingin meminta agar fitur chat diaktifkan pada Cheat Engine. Apakah memungkinkan untuk mengaktifkan atau menambahkan fitur chat engine di versi berikutnya? Fitur ini akan sangat membantu dalam berkomunikasi dan berdiskusi langsung di dalam aplikasi.

Terima kasih sebelumnya atas perhatiannya!

**Comment by C4ft4w** [comment]
Halo tim Cheat Engine,


Saya ingin mengajukan permintaan untuk mengaktifkan atau menambahkan fitur chat engine pada aplikasi Cheat Engine. Fitur ini akan sangat membantu untuk berkomunikasi dan berdiskusi langsung dalam aplikasi, terutama saat kolaborasi atau troubleshooting.


Terima kasih atas perhatian dan kerjasamanya!

**Comment by C4ft4w** [comment]
Halo tim pengembang,

**Comment by C4ft4w** [comment]
Saya ingin mengajukan permintaan untuk mengaktifkan fitur chat engine pada aplikasi ini. Fitur ini menurut saya akan sangat membantu untuk berkomunikasi atau berdiskusi langsung di dalam aplikasi.

**Comment by C4ft4w** [comment]
Mohon dipertimbangkan untuk diaktifkan pada update berikutnya.

**Comment by C4ft4w** [comment]
Terima kasih atas perhatian dan kerja samanya!
